README.md

```diff
@@ -0,0 +1,39 @@
+# Brisk
```

Comment:
@wokalski See if we need something else: [README.md](https://github.com/briskml/brisk/blob/feature/upgrade-infra/README.md).

We will also need a literate native speaker to proof-read, 'cause I'm probably bullshitting there.